empress/reconcile/median.py

```diff
@@ -68,14 +68,14 @@ def mapping_node_sort(ordered_gene_node_list, ordered_species_node_list, mapping
     return sorted_list
 
 
-def generate_scores(preorder_mapping_node_list, dtl_recon_graph, gene_root, normalize=True):
+def generate_frequencies_dict(preorder_mapping_node_list, dtl_recon_graph, gene_root, normalize=True):
     """
     Computes frequencies for every event
     :param preorder_mapping_node_list: A list of all mapping nodes in DTLReconGraph in double preorder
     :param dtl_recon_graph: The DTL reconciliation graph that we are scoring
```

Comment:
can you edit this comment to say sth like ":param dtl_recon_graph: The DTL reconciliation graph whose events we want to annotate with their corresponding frequencies"